### PR TITLE
Add animations to the fa-envelope

### PR DESCRIPTION
**Pull request by Edward-Kabue**
**Collaborated with Salim on:**
- Pair programming
- Adding animations to his code

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -98,7 +112,7 @@ p {
 }
 
 h1 {
-  font-family: 'Crete Round', sans-serif;
+  font-family: "Crete Round", sans-serif;
```
Comment (AndreLau11) [inline style.css:115]: you might consider using movement with the width of 100 right, as on the examples so your website can seem more interactive

File: style.css [hunk 2]
```diff
@@ -80,6 +80,20 @@ p {
...
+@keyframes spin {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
```
Comment (maen1980) [inline style.css:96]: good job, I think to add 50% between from and to make the animation more smooth

File: style.css [hunk 3]
```diff
@@ -5,7 +5,7 @@
   box-sizing: border-box;
 }
```
Comment (AndreLau11) [inline style.css:6]: You can consider adding 

- [ ] an Active action so when you give it a click it reacts too.
- [ ] or maybe a change of background color, according to your sigma model selection.

**Review by maen1980 — COMMENTED** [review]
good job.
please check the optional comment below and you are done 😀

**Review by AndreLau11 — COMMENTED** [review]
please check the notes below

**Pull request closed without merging**